examples/error_printer.cpp

```diff
@@ -104,6 +104,63 @@ namespace
 
 	inline constexpr auto divider =
 		"################################################################################"sv;
+
+	void print(std::string_view str)
+	{
+		// clang-format off
+		constexpr std::string_view control_char_escapes[] =
+		{
+			"\\u0000"sv,
+			"\\u0001"sv,
+			"\\u0002"sv,
+			"\\u0003"sv,
+			"\\u0004"sv,
+			"\\u0005"sv,
+			"\\u0006"sv,
+			"\\u0007"sv,
+			"\\b"sv,
+			"\\t"sv,
+			"\\n"sv,
+			"\\u000B"sv,
+			"\\f"sv,
+			"\\r"sv,
+			"\\u000E"sv,
+			"\\u000F"sv,
+			"\\u0010"sv,
+			"\\u0011"sv,
+			"\\u0012"sv,
+			"\\u0013"sv,
+			"\\u0014"sv,
+			"\\u0015"sv,
+			"\\u0016"sv,
+			"\\u0017"sv,
+			"\\u0018"sv,
+			"\\u0019"sv,
+			"\\u001A"sv,
+			"\\u001B"sv,
+			"\\u001C"sv,
+			"\\u001D"sv,
+			"\\u001E"sv,
+			"\\u001F"sv,
+		};
```

Comment:
This list was copied from forward_declarations.hpp, of course! @marzer Would it be OK if the example would directly use `impl::control_char_escapes` instead? I guess not, because it's an implementation detail. Just asking anyway  🤷  https://github.com/marzer/tomlplusplus/blob/2ca7ac6c2cf1ed91aa1134dc663b6601d5e4c545/include/toml%2B%2B/impl/forward_declarations.hpp#L127-L161

Reply:
I'm not fussed to be honest. That particular example isn't really intended as a code example, but more of an example of how the error messages themselves are formatted.

Reply:
OK, so would you prefer "error_printer.cpp" to use `control_char_escapes` from "impl/forward_declarations.hpp"? Or you you mean it doesn't really matter to you?

Reply:
I mean if you want to simplify it by re-using the implementation detail bit, rather than copying it, go ahead, I'm not going to be a purist about it.

Reply:
Hereby [force-pushed](https://github.com/marzer/tomlplusplus/compare/ffa0d3fecdd8106a6c0e7a1af1b224512d7fcee8..ac52b9e521d016992f21120e770268a2d7aaaa5d), please check!

Honestly I _am_ a bit of a purist on trying to avoid repetitive code, trying to follow the DRY principle 😇